packages/system/victoria-metrics-operator/Makefile

```diff
@@ -1,21 +1,14 @@
 NAME=victoria-metrics-operator
-NAMESPACE=cozy-victoria-metrics-operator
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
 	# VictoriaMetrics operator
 	helm repo add vm https://victoriametrics.github.io/helm-charts/
 	helm repo update vm
-	helm pull vm/victoria-metrics-operator --untar --untardir charts
+	helm pull vm/$(NAME) --untar --untardir charts
 	# Prometheus CRDs
 	helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 	helm repo update prometheus-community
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.